.github/workflows/zephyr.yml

```diff
@@ -53,6 +53,6 @@ jobs:
       - name: build
         run: docker run -v "$(pwd)/workspace":/workspace
              --workdir /workspace
-             ghcr.io/zephyrproject-rtos/zephyr-build:latest
+             ghcr.io/zephyrproject-rtos/zephyr-build:v0.24.2
```

Comment:
I understood this will come "for free" to `latest` in the next couple hours.

Reply:
@marc-hb Sooner or later yes, but as the process seem to be move Zephyr main before this image is necessarily update, it would seem more prudent for us to use an explicit version. We'll anyways bump the Zephyr version with an explicit manifest update (with a SOF PR), so maybe we should do (or not do) the SDK image update in the same go.

Reply:
> but as the process seem to be move Zephyr main before this image is necessarily update

No, 0.15.0 was the exception.

> it would seem more prudent for us to use an explicit version. We'll anyways bump the Zephyr version with an explicit manifest update (with a SOF PR), so maybe we should do (or not do) the SDK image update in the same go.

The most recent build failures in CI were not a bug, they were a feature as they gave us a good signal that 0.15.0 was exceptionally backwards incompatible. There was an announcement on a mailing list that no one noticed.

As most other things manual, I'm concerned SDK updates will happen in CI only when something breaks, which means CI will stop detecting issues with recent SDKs before developers do.